services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/validation/ValidationError.java

```diff
@@ -80,6 +80,7 @@ public int hashCode() {
     INVALID_URL,
     INVALID_PARTITIONING,
     TOO_MANY_DECIMAL_PLACES,
-    MISSING_ENTRY
+    MISSING_ENTRY,
+    UNKOWN_ENTRY
```

Comment:
Is this error type actually being used somewhere?

Reply:
Does not look like it.